File: API/Dtos/ComicForFindAuthorDto.cs
namespace API.Dtos
{
    public class ComicForFindAuthorDto
    {
        public int Id { get; set; }
        public string ComicName { get; set; }
        public string MainImage { get; set; }
        public int NOViews { get; set; }
        public int NOFollows { get; set; }
        public int NOComments { get; set; }
        public List<ChapterForFindAuthorDto> Chapters { get; set; }
    }
}

File: client/src/app/_models/comicForFindAuthorDto.ts
import { ChapterForFindAuthorDto } from "./chapterForFindAuthorDto";

export interface ComicForFindAuthorDto {
  id: number;
  comicName: string;
  mainImage: string;
  noViews: number;
  noFollows: number;
  noComments: number;
  chapters: ChapterForFindAuthorDto[]
}
